### DISCUSSION TITLE
Can I pass guard name to the middleware like `middleware('permission:publish-articles,web')`

### DISCUSSION THREAD

**Discussion by amoktar** [spatie/laravel-permission#2482] [Q&A] [answered]
Hello , is it possible to pass the guard name to the middleware ?

```php
Route::group(['middleware' => ['permission:publish-articles']], function () {
    //
});

Route::middleware('permission:publish-articles,web')->group(function () {
    //
});
```

Thanks

**Comment by drbyte** [comment]
Yes. How many guards do you have in your application?

**Reply by amoktar** [reply]
@drbyte Thanks for response and for this greate package 

I'm having the defaults (web and api)

**Reply by drbyte** [reply, marked as answer]
Yes you may pass the guard, as `,web` like in your example.

**Reply by parallels999** [reply]
Missing documentation on this?

**Reply by drbyte** [reply]
It is now documented.
